bin/scriptLib

```diff
@@ -18,7 +18,7 @@ if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
 
     echo "Using Akka SNAPSHOT ${AKKA_VERSION}"
 
-    EXTRA_OPTS="-Dakka.version=${AKKA_VERSION}"
+    EXTRA_OPTS="-Dakka_version=${AKKA_VERSION}"
```

Comment:
We just need to not use `akka.version` when reading the version from a `-D` property. 

This is overriding the existing akka property and causing the docs test to fail. 

That happens because `sbt-web` is using akka 2.3.16 and we override that property with the daily snapshot. 

On start-up the actor system fails because it detects that the value of `akka.version` is not the expected one.

Reply:
Oh, so `akka.version` is used internally by Akka. Good to know.